app/models/store.rb

```diff
@@ -29,6 +29,10 @@ def update_status(status)
     update(status: status)
   end
 
+  def self.all_active_stores
```

Comment:
With the use of enums, we actually have the Store Class Method .active, which would do what you've defined with this method.

Reply:
Great thought. I didn't consider using the enums in that way. I'll make the changes now.